react-native.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    project: {
+      ios: {},
+      android: {},
+    },
+    assets: ["./assets/fonts/"],
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/Airbenders-490/AirbendersFE/pullRequest?prid=8247201)
```suggestion
    assets: ['./assets/fonts/'],
```